README.md

```diff
@@ -70,13 +67,54 @@ python
 ./scripts/x-python-hooks.sh
 ```
 
+## install
+
+```bash
+./scripts/update.sh
+```
+
+## examples
+
+To use examples copy the best fit into the root of your project and edit as needed.
+
+all hooks:
+
+```bash
+cp pre-commit-config-all.yaml .pre-commit-config.yaml
```

Comment:
Could be simplified to just a list (or even a table) of available examples - the `cp` commands don't take into account relative paths so it wouldn't make sense for someone to run them as given anyway. I'm thinking something like

```md
- All hooks: [pre-commit-config-all.yaml](./pre-commit-config-all.yaml)
- Golang hooks: [pre-commit-config-go.yaml](./pre-commit-config-go.yaml)
- Python hooks: [pre-commit-config-python.yaml](./pre-commit-config-python.yaml)